config/kubernetes/vector-daemonset.yaml

```diff
@@ -0,0 +1,85 @@
+# Everything related to vector will live under the `telemetry` namespace.
```

Comment:
Any reason this is here? Wouldn't it make more sense to move this into the `distribution` folder?

Reply:
Its more of a config than a distribution, this may not be the right place but we probably want it somewhere. It also contains the embedded example config.

Reply:
The `/distribution` folder is full of configs that integrate with other platforms (docker, deb, rpm, systemd, etc). I would prefer that this live in `/distribution` since I believe it makes more sense there.